packages/boilerplate/composables/src/getters/index.ts

```diff
@@ -0,0 +1,14 @@
+// TODO: Create getters import.
+import * as productHelpers from './productHelpers';
+import * as cartHelpers from './cartHelpers';
+import * as categoryHelpers from './categoryHelpers';
+import * as checkoutHelpers from './checkoutHelpers';
+import * as userHelpers from './userHelpers';
+
+export default {
```

Comment:
With default export, there is no tree-shaking here. Please change the lines above as an example:
```ts
export * as userHelpers from './userHelpers';
```